icu4c/source/data/brkitr/rules/line_phrase_cj.txt

```diff
@@ -0,0 +1,379 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+# Copyright (c) 2002-2016  International Business Machines Corporation and
```

Comment:
Remove these two line of IBM copyright since this is a new file

Reply:
Resolved